llvm/include/llvm/Analysis/DXILResource.h

```diff
@@ -358,17 +360,18 @@ class ResourceInfo {
 private:
   ResourceBinding Binding;
   TargetExtType *HandleTy;
+  StringRef Name;
```

Comment:
The best reason I can see to include this here and not as part of the ResourceBinding member is to make it more readily accessible to `getName`, but I don't see that as being called at all? The other use of the name is in #140982 where it could be simplified as being part of the `Binding` compare instead of another parameter to `std::tie`. It works either way though.